datasets/nusax_sentiment/nusax_sentiment.py

```diff
@@ -0,0 +1,123 @@
+from datalabs.utils.download_manager import DownloadManager
+
+_CITATION = """\
+  @article{winata2022nusax,
+    title={NusaX: Multilingual Parallel Sentiment Dataset for 10 Indonesian Local Languages},
+    author={Winata, Genta Indra and Aji, Alham Fikri and Cahyawijaya, Samuel and Mahendra, Rahmad and Koto, Fajri and Romadhony, Ade and Kurniawan, Kemal and Moeljadi, David and Prasojo, Radityo Eko and Fung, Pascale and others},
+    journal={arXiv preprint arXiv:2205.15960},
+    year={2022}
+    }
+"""
+
+_DESCRIPTION = """\
+  NusaX is the first-ever parallel resource for 10 low-resource languages in Indonesia.
+  This dataset covers the sentiment analysis task.
+"""
+
+_HOMEPAGE = "https://github.com/IndoNLP/nusax"
+
+_LICENSE = """
+  CC BY-SA 4.0
+  Attribution — You must give appropriate credit, provide a link to the license, and indicate if changes were made. You may do so in any reasonable manner, but not in any way that suggests the licensor endorses you or your use.
+  ShareAlike — If you remix, transform, or build upon the material, you must distribute your contributions under the same license as the original.
+  No additional restrictions — You may not apply legal terms or technological measures that legally restrict others from doing anything the license permits.
+"""
+
+class NusaXSentimentConfig(datalabs.BuilderConfig):
+    """BuilderConfig for NusaX Sentiment Config"""
+
+    def __init__(self, **kwargs):
+        """BuilderConfig for NusaX Sentiment Config.
+        Args:
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(NusaXSentimentConfig, self).__init__(**kwargs)
+
+
+class NusaXSentiment(datalabs.GeneratorBasedBuilder):
+    """NusaX Sentiment dataset -- Multilingual Parallel Sentiment Dataset for 10 Indonesian Local Languages"""
+
+    BUILDER_CONFIGS = list(
+        [
+            NusaXSentimentConfig(
+                name=f"{l}",
+                version=datalabs.Version("1.0.0"),
+                description=f"NusaX: Multilingual Parallel Sentiment Dataset for 10 Indonesian Local Languages, {l} split",
+            )
+            for l in ["acehnese", "balinese", "banjarese", "buginese", "english", "indonesian", "javanese", "madurese", "minangkabau", "ngaju", "sundanese", "toba_batak"] 
+        ]
+    )
+    DEFAULT_CONFIG_NAME = "nusax_sentiment_indonesian"
+    _URL = "https://raw.githubusercontent.com/IndoNLP/nusax/main/datasets/sentiment"
+
+    def _info(self):
+
+        return datalabs.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datalabs.Features(
+                {
+                    "input": datalabs.Value("string"),
+                    "label": datalabs.ClassLabel(
+                        names=["positive", "neutral", "negative"]
+                    ),
+                }
+            ),
+            supervised_keys=None,
+            homepage=_HOMEPAGE,
+            license=_LICENSE,
+            citation=_CITATION,
```

Comment:
Thanks a lot for adding this dataset, it looks very good. Just one minor comment: it would be nice if you could add the field `task_templates` into `DatasetInfo`, here is one example: https://github.com/ExpressAI/DataLab/blob/ff755cbb0f7cdc8e373e1f7c12ef34e7551745d1/datasets/sst2/sst2.py#L70

Reply:
thanks for the comment @pfliu-nlp I added the task templates to the DatasetInfo.

Reply:
thanks a lot!!